parent/pom.xml

```diff
@@ -139,7 +139,7 @@
                     <version>1.6.8</version>
                     <configuration>
                         <serverId>ossrh</serverId>
-                        <nexusUrl>https://oss.sonatype.org/</nexusUrl>
+                        <nexusUrl>https://jakarta.oss.sonatype.org/</nexusUrl>
```

Comment:
Nit.  But, to be consistent with the previous usage in the pom, should these references be changed to properties as well?  We could have a property for the top level url and then re-use that for the sub-directory references.